pom.xml

```diff
@@ -197,24 +192,24 @@
           <groupId>org.pitest</groupId>
           <artifactId>pitest-maven</artifactId>
           <version>${pitest.maven}</version>
-          <executions>
-            <execution>
-              <id>pit-report</id>
-              <phase>test</phase>
-              <goals>
-                <goal>mutationCoverage</goal>
-              </goals>
-            </execution>
-          </executions>
-          <configuration>
-            <targetClasses>
-              <param>com.carhouse.*</param>
-            </targetClasses>
-            <targetTests>
-              <param>com.carhouse.*</param>
-            </targetTests>
-            <testPlugin>junit5</testPlugin>
-          </configuration>
+<!--          <executions>-->
```

Comment:
Commented?
No way!

Reply:
I've forgotten to fix it
I used this to speed up the build
I'll fix it